core/platform/models.py

```diff
@@ -113,59 +150,128 @@ def import_email_services(cls):
 
     @classmethod
     def import_memcache_services(cls):
+        """Imports and returns gae_memcache_services.
+
+        Returns:
+            module. The gae_memcache_services module.
+        """
         from core.platform.memcache import gae_memcache_services
         return gae_memcache_services
 
     @classmethod
     def import_taskqueue_services(cls):
+        """Imports and returns gae_taskqueue_services module.
+
+        Returns:
+            module. The gae_taskqueue_services module.
+        """
         from core.platform.taskqueue import gae_taskqueue_services
         return gae_taskqueue_services
 
     @classmethod
     def import_search_services(cls):
+        """Imports and returns gae_search_services module.
+
+        Returns:
+            module. The gae_search_services module.
+        """
         from core.platform.search import gae_search_services
         return gae_search_services
 
     NAME = 'gae'
 
 
 class Registry(object):
+    """Platform-agnostic interface for retrieving platform-specific modules.
+    """
+
+    # Maps platform names to the corresponding module registry classes.
     _PLATFORM_MAPPING = {
         _Gae.NAME: _Gae,
     }
 
     @classmethod
     def _get(cls):
+        """Returns the appropriate interface class for platform-specific
+        imports.
+
+        Returns:
+            model. _Gae Model.
+        """
         return cls._PLATFORM_MAPPING.get(feconf.PLATFORM)
 
     @classmethod
     def import_models(cls, model_names):
+        """Imports and returns the storage modules listed in model_names.
+
+        Args:
+            model_names: list(str). List of storage modules.
+
+        Returns:
+            list(module). The corresponding storage-layer modules.
+        """
         return cls._get().import_models(model_names)
 
     @classmethod
     def import_current_user_services(cls):
+        """Imports and returns platform-specific current_user_services
```

Comment:
This and the following docstrings should be consistent with each other. Either include "platform-specific" in all of them, or leave that adjective out.